Makefile

```diff
@@ -15,7 +15,7 @@ makel.mk:
 		curl \
 		--fail --silent --show-error --insecure --location \
 		--retry 9 --retry-delay 9 \
-		-O https://gitlab.petton.fr/DamienCassou/makel/raw/v0.6.0/makel.mk; \
+		-O https://github.com/damienCassou/makel/raw/v0.8.0/makel.mk; \
```

Comment:
damienCassou => DamienCassou

Both work but I always use the later.

Reply:
Strange, I'm sure I just copied it from my browser.

Reply:
I found a place in a repo where the URL was written with the wrong capitalization. I guess you just clicked on this link :-). I changed it now.

You fixed the URL in the wrong commit though: the first commit changes the URL to the bad version and the second fixes the first commit. Would you mind modifying the first commit?

Reply:
Right you are. I assumed pushing the correction in a second commit would be sufficient. I'm using all these PRs as an exercise to learn git (or magit) and working with forges like Github. How do I modify the first commit since the correction is already in my working tree and committed with the second commit? Squash both commits into one and do a `--force with lease` push?

Reply:
Squashing everything is the simplest option. But if you are on a learning journey, I instead suggest using `rebase --interactive` to "edit" the first commit. When rebase stops at the first commit, cherry pick what you want from the second commit and amend. Then, you can use `rebase --continue` to finish the rebase. I can give more details if something isn't clear.